gridstatus/ercot.py

```diff
@@ -1926,7 +1928,7 @@ def _make_bid_curve(df):
             else:
                 df_offers_hourly = (
                     df_offers.groupby(["Delivery Date", "Hour Ending"])
-                    .apply(_make_bid_curve)
+                    .apply(_make_bid_curve, include_groups=False)
```

Comment:
https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.core.groupby.DataFrameGroupBy.apply.html

> Deprecated since version 2.2.0: Setting include_groups to True is deprecated. Only the value False will be allowed in a future version of pandas.